### PR TITLE
David radar readme  update

### PR DESCRIPTION
**Pull request by Davidohis**
Updating  radar readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,4 +33,19 @@ $ npm run start
 $ npm run generate
 ```
 
-For detailed explanation on how things work, check out [Nuxt.js docs](https://nuxtjs.org).
```
Comment (Shark) [inline README.md:22]: Could you remove the lines starting from "# build for production" until "$ npm run generate"? Those are not really useful for someone new to the project and the build is documented in the [GitHub workflow](https://github.com/TechLabs-Berlin/radar/blob/master/.github/workflows/deploy.yml#L31) anyway.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+
+- [TechLabs Semester Radar Page ](https://bln.techlabs.org/timeline) 
+- [Slack Channel](https://techlabs-community.slack.com/) 
+
+
+## About Radar
+> The Radar is built as a static single page app (SPA) with Nuxt.js using Vue.js as the frontend framework.
+  For detailed explanation on how things work, check out [Nuxt.js docs](https://nuxtjs.org).
+
+ ![Radar Project](assets/radar-image.png)
```
Comment (Shark) [inline README.md:15]: I'd appreciate it if you could scale down the screenshot down to something like 500px. A 600k file referenced in the README to show off the page seems a bit much 😄

File: README.md (hunk 1 above)
Comment (Davidohis) [reply README.md:22]: Okay sir

**Comment by Davidohis** [comment]
I just fixed the project screen shot and build setup

**Review by Shark — APPROVED** [review]
Thanks David for your contribution!
I'll be in touch via email 🙂

**Pull request merged**